sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/MapsRenderAsyncClient.java

```diff
@@ -175,10 +176,9 @@ Mono<Response<BinaryData>> getMapTileWithResponse(MapTileOptions options, Contex
                 options.getTileIndex(), options.getTimestamp(), options.getMapTileSize(), options.getLanguage(),
                 options.getLocalizedMapView(), context)
             .onErrorMap(throwable -> {
-                if (!(throwable instanceof ErrorResponseException)) {
+                if (!(throwable instanceof ErrorResponseException exception)) {
```

Comment:
Why do we need `exception` here? Other SDKs don't have `exception` in the statement.

Reply:
@dubiety it was suggested from Java